web/libs/editor/src/components/BottomBar/Controls.jsx

```diff
@@ -60,7 +59,7 @@ export const Controls = controlsInjector(
         if (addedCommentThisSession) {
           selected?.submissionInProgress();
           callback();
-        } else if ((currentComment ?? "").trim()) {
+        } else if (Object.keys(currentComment).length > 0) {
```

Comment:
I think, we should check only current comment for current annotation. The rest could still stay there

Reply:
Thanks @hlomzik, I miss understood the way that we were validating this! It's fixed!